fix(rdb): block low latency (#2505)
scaleway/docs-content: managed-databases/postgresql-and-mysql/how-to/change-volume-type.mdx

    The Database Instance goes into the `Initializing` state.

    <Message type="important">

>       If you are changing from a Block to a local volume, make sure your current node type has a local storage size greater than or equal to your current Block Storage size first. If this is not the case, you must [upgrade your current node type](/managed-databases/postgresql-and-mysql/how-to/upgrade-a-database/) to one with enough local storage before you change the volume type.

Reviewer: ```suggestion
      If you are changing from a Block to a local volume, make sure to select a node with  a local storage size greater than or equal to your current Block Storage.
```
